harry-runner/src/harry/runner/HarryRunner.java

```diff
@@ -59,37 +53,7 @@ public void runWithInJvmDtest() throws Throwable
         ScheduledThreadPoolExecutor executor = new ScheduledThreadPoolExecutor(1);
         executor.setRemoveOnCancelPolicy(true);
 
-        Configuration.ConfigurationBuilder configuration = new Configuration.ConfigurationBuilder();
-
-        long seed = System.currentTimeMillis();
-        configuration.setSeed(seed)
-                     .setSUT(new InJvmSut.InJvmSutConfiguration(3,
-                                                                10,
-                                                                System.getProperty("harry.root", "/tmp/cassandra/harry/") + System.currentTimeMillis()))
-                     .setPartitionDescriptorSelector(new Configuration.DefaultPDSelectorConfiguration(10, 100))
-                     .setClusteringDescriptorSelector((builder) -> {
-                         builder.setNumberOfModificationsDistribution(new Configuration.ConstantDistributionConfig(10))
-                                .setRowsPerModificationDistribution(new Configuration.ConstantDistributionConfig(10))
-                                .setMaxPartitionSize(100)
-                                .setOperationKindWeights(new Configuration.OperationKindSelectorBuilder()
-                                                         .addWeight(OpSelectors.OperationKind.DELETE_ROW, 1)
-                                                         .addWeight(OpSelectors.OperationKind.DELETE_COLUMN, 1)
-                                                         .addWeight(OpSelectors.OperationKind.DELETE_RANGE, 1)
-                                                         .addWeight(OpSelectors.OperationKind.WRITE, 97)
-                                                         .build());
-                     })
-                     .setRowVisitor(new Configuration.DefaultRowVisitorConfiguration())
-                     .setClock(new Configuration.ApproximateMonotonicClockConfiguration((int) TimeUnit.HOURS.toSeconds(2) + 100,
-                                                                                        1,
-                                                                                        TimeUnit.SECONDS))
-                     .setRunTime(2, TimeUnit.HOURS)
-                     .setCreateSchema(true)
-                     .setTruncateTable(false)
-                     .setDropSchema(false)
-                     .setModel(ExhaustiveChecker::new)
-                     .setRunner(new Configuration.ConcurrentRunnerConfig(1, 1, 1));
-
-        Runner runner = configuration.build().createRunner();
```

Comment:
all the config above will come from the YAML files now

Reply:
This is great. I should've done it this way in the first place, but this code existed long before there was even any configuration. Great job!